browser/src/VariantPage/PopulationsTable.tsx

```diff
@@ -66,26 +76,29 @@ const renderPopAF = (af: number) => {
   return af.toPrecision(4)
 }
 
-type OwnPopulationsTableProps = {
-  columnLabels?: {
-    ac?: string
-    an?: string
-    af?: string
-  }
-  populations: {
+type Population = {
+  id: string
+  name: string
+  ac: number
+  an: number
+  ac_hemi?: number
+  ac_hom?: number
+  subpopulations?: {
     name: string
     ac: number
     an: number
     ac_hemi?: number
     ac_hom?: number
-    subpopulations?: {
-      name: string
-      ac: number
-      an: number
-      ac_hemi?: number
-      ac_hom?: number
-    }[]
   }[]
+}
+
+type OwnPopulationsTableProps = {
```

Comment:
while we're in here let's get rid of the `Own` here and call this `PopulationTableProps` or even just `Props` if it's not ambiguous. IIRC the `Own` that crops up in lots of names of types of component properties is an artifact of how the automatic ts-migrate script converted Proptypes, neither of which is relevant anymore.